Add post about remote work (#577)
ePages-de/epages-devportal: _posts/2019/2019-12-17-what-working-with-a-distributed-team-taught-us-about-agile-team-collaboration.md

---
layout: post
title: What working with a distributed team taught us about agile team collaboration
date: 2019-12-17
header_image: public/remote-work.jpg
header_position: top
header_overlay: true
category: methods-and-tools
tags: ["scrum", "retro", "agile", "remote"]
authors: ["Stefanie"]
about_authors: ["sholler"]
---

One of our teams suddenly spread over four locations including onboarding two external developers. Before, it was a two location team, that met regularly for sprint change.

> In the beginning, one of our developers, who was already working remote for a longer time, gave me the following advice: Never try to make things work the way they had been working for a colocated team, but to figure out alternative ways. 

Reviewer: ```suggestion
In the beginning, one of our developers, who was already working remote for a longer time, gave me the following advice: Never try to make things work the way they had been working for a co-located team, but to figure out alternative ways. 
```

In that very moment, I felt overwhelmed by this new situation. 
Remote collaboration felt like a huge constraint: You can’t do group work or ice breakers in which the team has to move around. 
You can’t even go for lunch with your team or meet at the water dispenser for a casual chat. 

## Start by questioning every practice you have whether it is suitable for remote purposes

His advice resonated with me and I started to see this a challenge and started looking for new ways of cooperation. 
The first thing I wanted to change was the retrospective. 

> Before, we had each team sitting in a conference room in their location and connect remotely via hangouts, or the team members from our location within Germany travelled to our office. However, now I had to deal with 3-4 conference rooms and/or remote team members at once. 

Reviewer: ```suggestion
Before, we had each team sitting in a conference room in their location and connect remotely via hangouts, or the team members from our location within Germany traveled to our office. However, now I had to deal with 3-4 conference rooms and/or remote team members at once. 
```

> It seemed impossible to discuss in the way we had before and to summarize results on a google document. 

Reviewer: Company names should be in upper case.

```suggestion
It seemed impossible to discuss in the way we had before and to summarize results on a Google document. 
```

Sitting in a conference room was not an option anymore. 
Even though it felt really awkward to sit in a room together while meeting in a video call, this was the most effective change to our discussion culture. 
Suddenly everyone was equal and participation possibilities were equally distributed, too. 
We could see all of the team members individually, could better get their mood by reading their face expressions and most importantly, I could see when someone unmuted, because she wanted to say something and react to that. 
In the conference room setting, it was difficult to impossible for me to get those small signals. 
At the same time, the team brought up the idea of a mini retrospective at the end of every meeting. 
It took just five minutes and provided really valuable input for me. 
The team members gave feedback on whether they like or disliked the new online tools we used, whether they wanted more or less strict moderation, and especially also gave feedback on how the sound quality was.

## Improve your sound quality

Everyone who works with distributed teams knows about this and I can already hear your sighs. 

> We had all the issues from bad microphone quality (so you could barely hear what your colleagues were saying) to Google Hangouts  automatically levelling the sound level of your microphones. 

Reviewer: ```suggestion
We had all the issues from bad microphone quality (so you could barely hear what your colleagues were saying) to Google Hangouts  automatically leveling the sound level of your microphones. 
```

> This was and still is my least favourite part about working in a distributed team, but the time spent was definitely worth it. 

Reviewer: ```suggestion
This was and still is my least favorite part about working in a distributed team, but the time spent was definitely worth it. 
```

We kept looking for solutions until we had fixed all technical issues. 
Annoying, but it worked. 
I had even set up a meeting with my PO, who is working in a different location that I am, just to try out different devices for  45 minutes. 
We also tried out things, that could affect sound quality (e.g. a laptop between you and the microphone), in order to make the teams sensible to what to watch out for. 
It definitely helped to be in the remote position from time to time to experience the limiting factors yourself. 
Another option is to record your meeting (e.g. the Standup) to become aware of sound quality issues and common patterns.

## Remote first as a principle

Early in our learning path towards a better remote collaboration, one of our developers recommend a podcast with Bryan Helming, Co-Founder of Zapir, a company that works exclusively remote. In this podcast, Bryan said, one of the company’s values they committed to was “Remote first”. 

> Like I had said in the beginning, examine every process on how you could make a physical process work for a distributed team.

Reviewer: "had said" is past perfect but we need present perfect here. See https://www.quora.com/Which-is-correct-have-said-or-had-said

```suggestion
Like I have said in the beginning, examine every process on how you could make a physical process work for a distributed team.
```

> I don’t want to bore you with the details, but besides banning all posters or physical processes we had in the team office, we also emphasised the already existing culture of using the team chat for all work related discussions and documenting all decisions in our Confluence space. 

Reviewer: American English is the official company language.

```suggestion
I don’t want to bore you with the details, but besides banning all posters or physical processes we had in the team office, we also emphasized the already existing culture of using the team chat for all work related discussions and documenting all decisions in our Confluence space. 
```

It also helps to consciously change your habit of just asking your closest colleague of the team to just picking someone from another location to ask for an advice or his opinion on a certain topic. 
Besides using Retrium for the retrospective , we also looked for a tool to make estimation more smoothly. 

> We chose www.planitpoker.de because it easily allows everyone to estimat and gives a nice visual overview of the outcome. 

Reviewer: Typo: estimat => estimate
Wrong URL: planitpoker.de => planitpoker.com

```suggestion
We chose [planITpoker](https://www.planitpoker.com) because it easily allows everyone to estimate and gives a nice visual overview of the outcome. 
```

> Retrium and planitpoker worked so well for us, that we sometimes also use them for colocated teams.

Reviewer: ```suggestion
Retrium and planITpoker worked so well for us, that we sometimes also use them for co-located teams.
```

Overall, distributed collaboration, that felt like a huge limiting constraint in the beginning, became a big chance to learn to adapt and push my ability to collaborate with the team further. Not that we are already at the end, there are still a lot of things to learn and improve for us. But I had the impression, that by learning to foster collaboration and trust in a distributed team, I also learned a lot on how to do this for collocated teams more effectively.

If you want to go further, here is a list of some of the tools and resources we found useful:

[Johanna Rothman, Mark Kilby: From chaos to successful distributed teams](https://www.amazon.com/Chaos-Successful-Distributed-Agile-Teams-ebook/dp/B07PRYM1TF)

Podcast with Bryan Helmig, Co-Founder of Zapier
https://www.se-radio.net/2019/05/episode-368-bryan-helmig-on-managing-distributed-teams/

[Future of Work Podcast with Wade Foster, CEO of Zapir]
(https://www.stitcher.com/podcast/the-future-of-work-podcast/e/59481731?autoplay=true)

www.retrium.com

> www.planitpoker.de 

Reviewer: The domain "www.plainitpoker.de" does not exist.

```suggestion
https://www.planitpoker.com
```

> For a really exhaustive list of ressources concerning remote work check out Lisette Sutherlands website for ressources:

Reviewer: Typo:

```suggestion
For a really exhaustive list of resources concerning remote work check out Lisette Sutherlands website for resources:
```
